core/src/main/java/org/elasticsearch/cluster/routing/RoutingService.java

```diff
@@ -90,52 +79,26 @@ protected void doStop() {
 
     @Override
     protected void doClose() {
-        FutureUtils.cancel(scheduledRoutingTableFuture);
```

Comment:
I am not sure its a real problem, cause on shutdown we cancel ongoing ones anyhow, but will add